{
  final GradleException exception=new GradleException("<message>");
  context.checking(new Expectations(){
{
      one(logger).error(with(containsString("Build failed with an exception.")));
      one(logger).error(String.format("Exception: %s",exception));
    }
  }
);
  reporter=new BuildExceptionReporter(logger);
  reporter.buildFinished(new BuildResult(null,exception));
}
